index.md

```diff
@@ -103,6 +103,17 @@ workshop is only open to people from a particular institution.
 {% include lc/who.html %}
 {% endif %}
 
+{% comment %}
+HUMAN LANGUAGE
+{% endcomment %}
+
+{% unless site.human_language == "" %}
+<p id="human-language">
+<strong>The course will be conducted in {{ site.human_language }}.</strong> For this course, we will expect our participants to have basic understanding of {{ site.human_language }}. We will have several helpers to assist you if you need some terms to be translated.
```

Comment:
course -> workshop?

Reply:
I'm not sure about `We will have several helpers to assist you if you need some terms to be translated.` Which terms do you mean?

Reply:
I think I copied this from @malvikasharan and I understood it as any word that could have been difficult to understand. How about replacing "terms" with "words"?

Reply:
> How about replacing "terms" with "words"?

it is better though it does not add clarity... I guess I don't understand the purpose of this sentence. I'd say something like this:

> Helpers will be available if you need assistance with ... understanding lesson material. (???)

I'd also change wrap the body of the paragraph.